lib/src/wallet.rs

```diff
@@ -199,7 +198,7 @@ pub struct LightWallet {
 use crate::wallet::traits::{Diversifiable as _, ReadableWriteable};
 impl LightWallet {
     pub fn serialized_version() -> u64 {
-        return 25;
+        return 26;
```

Comment:
Does this mean that existing `Zingo!` wallets will have to be reloaded?

Reply:
Nope! That's the whole point of incrementing the version number.. When we read a v25 wallet, we know it had anchors saved, so we read them and ignore them.